From b080dc5bd0dfc0b33c6cfc31f909c93d5e63c186 Mon Sep 17 00:00:00 2001
From: Jeff Crukley <jcrukley@gmail.com>
Date: Wed, 25 Jul 2018 15:05:01 -0400
Subject: [PATCH] ALSA: usb-audio: Add support for Encore mDSD USB DAC
Git-commit: b080dc5bd0dfc0b33c6cfc31f909c93d5e63c186
Patch-mainline: v4.19-rc1
References: bsc#1051510

This patch adds native DSD playback support for the Encore mDSD USB DAC by
specifying the vendor and product ID's

Signed-off-by: Jeff Crukley <jcrukley@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 06ae3f5adf4b..dde87d64bc32 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1373,6 +1373,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
 		break;
 
+	case USB_ID(0x16d0, 0x09dd): /* Encore mDSD */
 	case USB_ID(0x0d8c, 0x0316): /* Hegel HD12 DSD */
 	case USB_ID(0x16b0, 0x06b2): /* NuPrime DAC-10 */
 	case USB_ID(0x16d0, 0x0733): /* Furutech ADL Stratos */
-- 
2.19.2

